website/source/docs/builders/vmware-iso.html.markdown

```diff
@@ -108,6 +108,13 @@ builder.
     User's Guide](http://www.vmware.com/pdf/VirtualDiskManager.pdf) for desktop
     VMware clients. For ESXi, refer to the proper ESXi documentation.
 
+-   `disk_adapter_type` (string) - The adapter type of the VMware virtual disk
+    to create. The default is "lsilogic". This option is for advanced usage,
+    modify only if you know what you're doing. For more information, please
+    consult the [Virtual Disk Manager User's Guide]
+    (http://www.vmware.com/pdf/VirtualDiskManager.pdf) for desktop VMware
```

Comment:
In any case, I've updated the links to the Virtual Disk Manager User's Guide PDF to open in a new window and also added the Adobe PDF icon. Thank you.